code-challenges.test.js

```diff
@@ -33,19 +33,96 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+describe('divideByThree', () => {
+    it('takes an object as an argument and decides if the number inside it is evenly divisible by three or not.', () => {
+      const object1 = { number: 15 }
+      // Expected output: "15 is divisible by three"
+      const object2 = { number: 0 }
+      // Expected output: "0 is divisible by three"
+      const object3 = { number: -7 }
+      // Expected output: "-7 is not divisible by three"
+      expect(divideByThree(object1.number)).toEqual("15 is divisible by three")
+      expect(divideByThree(object2.number)).toEqual("0 is divisible by three")
+      expect(divideByThree(object3.number)).toEqual("-7 is not divisible by three")
+    })
+  })
+// ReferenceError: divideByThree is not defined
+
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+// Pseudo code: 
+// input: object
+// output: object with it's potential to be divided by 3.
```

Comment:
The pseudo code is unclear. What does this line mean? Try to make your pseudo code clear!